### PR TITLE
chore: bump dependency versions.

### PR DESCRIPTION
**Pull request by peterwht**

### CODE REVIEW

File: cypress.config.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default defineConfig({
   projectId: 'eup7bh',
   e2e: {
     specPattern: 'cypress/e2e/**/*.{js,jsx,ts,tsx}',
-    baseUrl: 'http://localhost:8081/',
```
Comment (Daanvdplas) [inline cypress.config.ts:11]: Why does this need to be changed?

File: vite.config.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export default defineConfig({
       cypress: true,
     }),
   ],
-  server: { port: 8081 },
```
Comment (Daanvdplas) [inline vite.config.ts:20]: And this?

File: snapshots.js [hunk 3]
```diff
@@ -1,8 +1,5 @@
-// Copyright 2022-2024 use-ink/contracts-ui authors & contributors
-// SPDX-License-Identifier: GPL-3.0-only
-
```
Comment (Daanvdplas) [inline snapshots.js:3]: Why is this removed?

**Review by AlexD10S — DISMISSED** [review]
Other than Daan's comments, everything looks good to me. I've tested it, and everything seems fixed.

File: vite.config.ts (hunk 2 above)
Comment (peterwht) [reply vite.config.ts:20]: for the CI

File: snapshots.js (hunk 3 above)
Comment (peterwht) [reply snapshots.js:3]: Thanks! Auto-generated file, will get the license back in

File: cypress.config.ts (hunk 1 above)
Comment (peterwht) [reply cypress.config.ts:11]: For the CI. Something about using localhost was causing the ui to be unreachable by cypress

**Review by Daanvdplas — APPROVED** [review]
